app/MetricUnitConverter.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace ConverterApp;
+
+class MetricUnitConverter extends Converter
+{
+    public function converter($currentUnit, $newUnit)
```

Comment:
This function accepts value and not unit. Their names should be adjusted accordingly.